net/lp_rpc.proto

```diff
@@ -160,6 +164,9 @@ message VideoProfile {
     MP4        = 1;
   }
   Format format = 21;
+
+  // FPS Denominator of VideoProfile
+  uint32 fps_den = 22;
```

Comment:
I think we've used a camelCase convention here so far; `fpsDen` is probably more consistent?

Reply:
I saw snake_style being used in all other protobuf structs except SegData. But as this is part of SegData I'll use camelCase here as well.